contracts/src/v0.8/vrf/dev/SubscriptionAPI.sol

```diff
@@ -405,14 +409,18 @@ abstract contract SubscriptionAPI is ConfirmedOwner, IERC677Receiver, IVRFSubscr
     if (consumers.length == MAX_CONSUMERS) {
       revert TooManyConsumers();
     }
-    mapping(uint256 => uint64) storage nonces = s_consumers[consumer];
-    if (nonces[subId] != 0) {
+    mapping(uint256 => ConsumerConfig) storage consumerConfigs = s_consumers[consumer];
+    ConsumerConfig consumerConfig = consumerConfigs[subId];
```

Comment:
Missing `storage` or `memory`. Both should cost the same gas since it's 1 storage slot, I think.

Reply:
thanks. updated